ios/brave-ios/Sources/Brave/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -466,7 +466,11 @@ public class SearchViewController: UIViewController, LoaderListener {
         }
         make.bottom.equalTo(view.safeArea.bottom).offset(-offset)
       } else {
-        make.bottom.equalTo(view).offset(-keyboardHeight)
+        var offset: CGFloat = 0
+        if keyboardHeight > 0 {
+          offset = keyboardHeight - view.safeAreaInsets.bottom
+        }
+        make.bottom.equalTo(view.safeArea.bottom).offset(-offset)
```

Comment:
Maybe its possible to just avoid keyboard height calculations and use https://developer.apple.com/documentation/uikit/uiview/keyboardlayoutguide instead?

Reply:
i tried `make.bottom.equalTo(view.keyboardLayoutGuide.snp.top)` it always `only` layout behind the keyboard for the first time. I have no idea why. 
tried calculate the keyboard height using keyboardLayoutGuide is always wrong too.